### DISCUSSION TITLE
1 minute plot creation time?

### DISCUSSION THREAD

**Discussion by cayolblake** [madMAx43v3r/chia-plotter#681] [Q&A]
Hello @madMAx43v3r 

What would it take to have one minute plot time?

Is it hypothetically possible? Would it require a GPU for parallel processing?

**Comment by alexwwstx** [comment]
if it's possible, I don't think the current Chia algorithm will not be changed

**Reply by cayolblake** [reply]
How/Why so?

**Reply by keliew** [reply]
Simply because plotting speed is just one of the many requirements to win a block.

https://www.youtube.com/watch?v=YREMKqXqRdE

**Reply by cayolblake** [reply]
> https://www.youtube.com/watch?v=YREMKqXqRdE

I'm talking about plot-creation-time.

Unless, you are talking about realtime plot creation will in-theory defy the practicality of using Chia blockchain as a whole, however, we're talking 1 minutes versus 30 seconds.

Let's say I'm asking about 10 minutes, not one :)

**Comment by XCHSystems** [comment]
Managed to get mine down to just over 25 minutes, will be trying a 24C CPU over the next few days, the following was on a Xeon 6226R 16 Core / 32T CPU, 64GB RAM, 2x Intel P5800X 800GB in a VROC RAID0, number of buckets for all phases was 256 as I have noticed that you get better performance due to the smaller block I/O size versus 128 buckets

Crafting plot 22 out of 100
Process ID: 136857
Number of Threads: 20
Number of Buckets P1:    2^8 (256)
Number of Buckets P3+P4: 2^8 (256)
Pool Public Key:   8c5221a1f3929fb7424d95db4324b20e9a04d1963fd5ff18a2616e8db43abcc4549dd94eeb8a095374c2037ac97c44c4
Farmer Public Key: 8f7490c4cf2674dd905186b77030e718f516e4f45b79c7388b58ddd4b5b6e51bbb11524204df3e287f75429df99a35d0
Working Directory:   /P5800XRAID/
Working Directory 2: /P5800XRAID/
Plot Name: plot-k32-2021-06-28-06-53-82bad2e3497a0c919c1b080db17424b55d75402f056052fcbd6fa2866d952aa4
[P1] Table 1 took 15.4555 sec
[P1] Table 2 took 96.8582 sec, found 4294945704 matches
[P1] Table 3 took 116.09 sec, found 4294862718 matches
[P1] Table 4 took 139.715 sec, found 4294824861 matches
[P1] Table 5 took 133.542 sec, found 4294609570 matches
[P1] Table 6 took 125.733 sec, found 4294194282 matches
[P1] Table 7 took 97.4183 sec, found 4293270629 matches
Phase 1 took 724.825 sec
[P2] max_table_size = 4294967296
[P2] Table 7 scan took 8.43278 sec
[P2] Table 7 rewrite took 32.2736 sec, dropped 0 entries (0 %)
[P2] Table 6 scan took 28.2031 sec
[P2] Table 6 rewrite took 46.6637 sec, dropped 581395367 entries (13.5391 %)
[P2] Table 5 scan took 27.0103 sec
[P2] Table 5 rewrite took 44.4681 sec, dropped 762087946 entries (17.7452 %)
[P2] Table 4 scan took 26.8178 sec
[P2] Table 4 rewrite took 44.0633 sec, dropped 828925733 entries (19.3006 %)
[P2] Table 3 scan took 26.5018 sec
[P2] Table 3 rewrite took 43.5128 sec, dropped 855058024 entries (19.9089 %)
[P2] Table 2 scan took 26.4861 sec
[P2] Table 2 rewrite took 43.3708 sec, dropped 865566557 entries (20.1531 %)
Phase 2 took 409.029 sec
Wrote plot header with 268 bytes
[P3-1] Table 2 took 28.5711 sec, wrote 3429379147 right entries
[P3-2] Table 2 took 23.7023 sec, wrote 3429379147 left entries, 3429379147 final
[P3-1] Table 3 took 35.492 sec, wrote 3439804694 right entries
[P3-2] Table 3 took 24.2656 sec, wrote 3439804694 left entries, 3439804694 final
[P3-1] Table 4 took 35.7539 sec, wrote 3465899128 right entries
[P3-2] Table 4 took 24.6493 sec, wrote 3465899128 left entries, 3465899128 final
[P3-1] Table 5 took 35.9743 sec, wrote 3532521624 right entries
[P3-2] Table 5 took 25.1719 sec, wrote 3532521624 left entries, 3532521624 final
[P3-1] Table 6 took 37.8774 sec, wrote 3712798915 right entries
[P3-2] Table 6 took 26.03 sec, wrote 3712798915 left entries, 3712798915 final
[P3-1] Table 7 took 32.6431 sec, wrote 4293270629 right entries
[P3-2] Table 7 took 31.0573 sec, wrote 4293270629 left entries, 4293270629 final
Phase 3 took 363.672 sec, wrote 21873674137 entries to final plot
[P4] Starting to write C1 and C3 tables
[P4] Finished writing C1 and C3 tables
[P4] Writing C2 table
[P4] Finished writing C2 table
Phase 4 took 39.1398 sec, final plot size is 108813891911 bytes
Total plot creation time was 1536.7 sec (25.6117 min)
Started copy to /Plotdisk02/plot-k32-2021-06-28-06-53-82bad2e3497a0c919c1b080db17424b55d75402f056052fcbd6fa2866d952aa4.plot
